pkgs/development/ocaml-modules/mparser/pcre.nix

```
@@ -0,0 +1,18 @@
+{ fetchFromGitHub, buildDunePackage, ocaml_pcre, mparser }:
+
+buildDunePackage rec {
+  pname = "mparser-pcre";
+  version = "1.3";
```

You can also reuse the version attr from mparser.

```suggestion
  inherit (mparser) version;
```